s3watcher/lambda/package.json

```diff
@@ -4,17 +4,20 @@
   "description": "",
   "main": "S3Watcher.js",
   "dependencies": {
+    "aws-sdk": "^2.2.21",
```

Comment:
should this be in dev deps? The aws sdk is already included in the lambda environment.